Dockerfile

```diff
@@ -1,24 +1,35 @@
-FROM node:6-stretch
+FROM node:6-stretch AS build
 LABEL maintainer="will.price94@gmail.com"
 LABEL version="0.0.1"
-RUN apt-get update -qq && \
-    apt-get install -y python
 # Prevent npm from spamming
+ENV NPM_CONFIG_LOGLEVEL=warn
 RUN npm config set loglevel=warn && \
     npm config set progress=false
 WORKDIR /app/
 COPY package.json package-lock.json ./
 RUN npm install
 COPY . .
-RUN npm run build
+RUN mkdir -p /build/client
```

Comment:
How about `mkdir -p /build/{client,server}` and drop the next mkdir

Reply:
done